.github/workflows/daily-build.yml

```diff
@@ -71,15 +71,11 @@ jobs:
               env:
                 JAVA_HOME: /usr/lib/jvm/default-jvm
                 JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
-            
-            # Publish Github Package
-            - name: Publish Github Package
```

Comment:
We need this for release publishing
@udda1996

Reply:
this is fine as we use central stage instead of GH packages

Reply:
Yes. We need to remove this publishing as github packages part.